index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>John Thiong'o Murianki</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap' rel='stylesheet'>
+</head>
+<body>
+    <div id ="content">
+    <div id ="status-bar"></div>
+    <header>
+        <a id="logo">
+            <img src="/media/My Logo.png">
+        </a>
+        <nav>
+            <div class="line"></div>
+            <div class="line"></div>
+            <div class="line"></div>
+        </nav>
+    </header>
+    <section id="greeting">
+        <p>I'm John Glad to see you</p>
+    </section>
+    <section id="about">
+        <p>
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+    </section>
+    <section id="socials">
+        <h2 id="connect">LET'S CONNECT</h2>
+        <div id="social-icons"> 
+            <img src="/media/social media icons.png">
```

Comment:
- [ ] As mentioned before please instead of using a `<img>` tag for the social icons, as a project requirement, you should create each of them. Please use `<ul>` tag to group them, and `<li>` per icon; each icon should have a link (`<a>` tag). Because apart from being semantically correct, the `<a>` tag will let you connect your social networks with the icons, using `href` :+1: